vignettes/wrangling-survey-responses.Rmd

```diff
@@ -105,7 +107,7 @@ submissions_df <- submissions_df %>%
 colnames(submissions_df)
 ```
 
-For very large forms, it might be useful to investigate on how to sort the columns alphabetically or by the order they appeared in the form. With the small form here, we can still individually select the columns to put them into a meaningful order:
+For very large forms, it might be useful to investigate on how to sort the columns alphabetically or by the order they appeard in the form. With the small form here, we can still individually select the columns to put them into a meaningful order:
```

Comment:
I might be mistaken, but is not `appeared` actually the correct form, as opposed to `appeard`?

Reply:
you are indeed correct!